### ISSUE TITLE
How to extract the last 10009 last images of the official tensorflow ImageNet split as stated in BYOL paper.

### ISSUE THREAD

**Issue by JoohyungLee0106** [deepmind/deepmind-research#406]
In Appendix D.1., the BYOL paper states that they took _the last 10009 last images of the official TensorFlow ImageNet split_ for the validation set to tune various hyperparameters.
I would like to reproduce the result and wonder how I can extract the last 10009 last images of the official TensorFlow ImageNet split. Thank you.
